src/helpers/updateHierarchy.helper.js

```diff
@@ -157,6 +167,24 @@ class HierarchyService {
       isFirstTime: true
     };
 
+    // Set textbook allow to open for contribution
+    response.content['openForContribution'] = true;
+    let chapterCountForContribution = 0;
+
+    _.forEach(response.content.children, (child, i) => {
+        let cindex = children.findIndex(item => item.id === child.identifier);
+
```

Comment:
Here, what I would ideally do it, if we are not going to use the `index`, I would rather use `_.includes`. 
Here it is not about right or wrong, just the readability.

Reply:
closing as discussed